airflow/www/static/js/cluster-activity/live-metrics/Health.tsx

```diff
@@ -107,11 +106,16 @@ const Health = (props: CenterProps) => {
               latestHeartbeat={data?.triggerer?.latestTriggererHeartbeat}
               mb={3}
             />
-            <HealthSection
-              title="Dag Processor"
-              status={data?.dagProcessor?.status}
-              latestHeartbeat={data?.dagProcessor?.latestDagProcessorHeartbeat}
-            />
+            {!!standaloneDagProcessor && (
```

Comment:
All item should have a margin bottom, but the last one. (Both when the last one is the `Dag Processor` or the `Triggerer`.

To not add ternary into the `mb` props you can simply define a margin top on the `Dag Processor` section and remove the `mb` on the Triggerer.

Reply:
@pierrejeambrun I've made these updates. Can you mark this a resolved? Let me know if further updates are needed.